### PR TITLE
Add authorizations to category resource #78

### LINKED ISSUE #78: Add authorizations to the category resource
Currently this should just lock it down to only allow users with the admin_categories flag set to manage them.

No one should have any other permissions.

### PR DESCRIPTION
**Pull request by grimm**
Fixes #78

### CODE REVIEW

**Comment by grimm** [comment]
@gina-alaska/awrvi Ready for review.

File: lib/tasks/dev.rake [hunk 1]
```diff
@@ -29,6 +29,26 @@ namespace :dev do
     install_phantomjs
   end
 
+  desc 'Toggle user permission for category admin'
+  task :toggle_category, [:email] => :environment do |_t, args|
```
Comment (teknofire) [inline lib/tasks/dev.rake]: this should be called category_admin and should work like the user_admin task.  IE. email is provided as a env var and if none is provided it picks the last user.

**Comment by teknofire** [comment]
I'm wondering if we should unify rake tasks for setting user permissions.

Maybe just make a dev:superuser task that sets all the permission flags to true for the given user

**Comment by smacfarlane** [comment]
@teknofire  I think for this pull request it's fine the way it is.   We will probably want to down the road namespace the toggle tasks and add a super in addition to the individual tasks.

**Comment by teknofire** [comment]
:+1:

**Comment by smacfarlane** [comment]
:+1:

**Pull request merged**